Jellyfin.HardwareVisualizer/Server/Database/PlatformType.cs

```diff
@@ -1,8 +1,10 @@
 namespace Jellyfin.HardwareVisualizer.Server.Database;
+using System.ComponentModel;
 
 public enum PlatformType
 {
 	Windows,
 	Linux,
+	[Description("Darwin")]
```

Comment:
that does not have the effect you would like to. You need to actually change the enum value and then create a new migration for it